x/cron/handler.go

```diff
@@ -0,0 +1,7 @@
+package cron
+
+import weave "github.com/iov-one/weave"
+
+func RegisterQuery(qr weave.QueryRouter) {
+	NewTaskResultBucket().Register("crontaskresults", qr)
```

Comment:
Maybe call the file `query.go` or merge this into `model.go`.

I was surprised to see `handler.go`, but this function is very important.

Reply:
I am also confused why this is always in the `handlers.go` :smile: Although I think it makes more sense to put it together with the bucket in the `model.go` file, all extensions declare it in `handlers.go`. I did the same for consistency.

Should I move it to `models.go`?

Reply:
Yeah, seems good.

I think I put it in handlers.go just to keep with the other global registration functions. But maybe not the best pattern

Reply:
Done in https://github.com/iov-one/weave/pull/850/commits/1ff70fdb2887b812e262cab9c4412da26f013ac3